src/pages/PreInterviewPage.tsx

```
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { AnswerInput, Button, Counter, Question, Timer } from "../components/common";
+import { ROUTES } from "../constants";
+import showToast from "../utils/toast";
+
+interface Question {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Question'. Consider to delete it or rename it.

'Question' is defined here:


(lint/suspicious/noRedeclare)